api/external_endpoint_test.go

```diff
@@ -76,6 +76,12 @@ func TestCreateExternalEndpointResult(t *testing.T) {
 			AuthorizationHeaderBearerToken: "Bearer token",
 			ExpectedCode:                   200,
 		},
+		{
+			Name:                           "good-token-success-true",
+			Path:                           "/api/v1/endpoints/g_n/external?success=true&error=failed",
+			AuthorizationHeaderBearerToken: "Bearer token",
+			ExpectedCode:                   200,
+		},
```

Comment:
It's a good test to have, but I would generally expect people to only send the `error` query parameter if `success` is `false`.
If possible, I'd like the name of the testing scenario to match that expectation and say that the error is ignored, i.e. `good-token-success-true-with-ignored-error-because-success-true`. It's a mouthful, but at least it's very clear.